specification/keyvault/Security.KeyVault.Certificates/routes.tsp

```diff
@@ -0,0 +1,609 @@
+op setCertificateContacts is KeyVaultOperation<
+  {
+    /**
+     * The contacts for the key vault certificate.
+     */
+    @body contacts: Contacts;
+  },
+  Contacts
+>;
+
+/**
+ * The GetCertificateContacts operation returns the set of certificate contact resources in the specified key vault. This operation requires the certificates/managecontacts permission.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+@summary("Lists the certificate contacts for a specified key vault.")
+@route("/certificates/contacts")
+@get
+op getCertificateContacts is KeyVaultOperation<{}, Contacts>;
+
+/**
+ * Deletes the certificate contacts for a specified key vault certificate. This operation requires the certificates/managecontacts permission.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+@summary("Deletes the certificate contacts for a specified key vault.")
+@route("/certificates/contacts")
+@delete
+op deleteCertificateContacts is KeyVaultOperation<{}, Contacts>;
+
+/**
+ * The GetCertificateIssuers operation returns the set of certificate issuer resources in the specified key vault. This operation requires the certificates/manageissuers/getissuers permission.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+#suppress "@azure-tools/typespec-azure-core/use-standard-names" "Operation name is already established"
+@summary("List certificate issuers for a specified key vault.")
+@route("/certificates/issuers")
+@get
+op getCertificateIssuers is KeyVaultOperation<
+  {
+    /**
+     * Maximum number of results to return in a page. If not specified the service will return up to 25 results.
+     */
+    @maxValue(25)
+    @minValue(1)
+    @query("maxresults")
+    maxresults?: int32;
+  },
+  CertificateIssuerListResult
+>;
+
+/**
+ * The SetCertificateIssuer operation adds or updates the specified certificate issuer. This operation requires the certificates/setissuers permission.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+#suppress "@azure-tools/typespec-azure-core/use-standard-names" "Operation name is already established"
+@summary("Sets the specified certificate issuer.")
+@route("/certificates/issuers/{issuer-name}")
+@put
+op setCertificateIssuer is KeyVaultOperation<
+  {
+    /**
+     * The name of the issuer. The value you provide may be copied globally for the purpose of running the service. The value provided should not include personally identifiable or sensitive information.
+     */
+    @path("issuer-name")
+    issuerName: string;
+
+    /**
+     * Certificate issuer set parameter.
+     */
+    #suppress "deprecated" "Property flattening is supported for legacy scenarios like Key Vault's"
+    @body
+    @flattenProperty
```

Comment:
This should go in the client.tsp file. Ideally. Because it really is a codegen switch.

Reply:
I opened an issue about this, because I agree, but unfortunately was told that this is a TSP limitation: https://github.com/Azure/typespec-azure/issues/1610